ironfish/src/wallet/scanner/walletScanner.ts

```diff
@@ -33,7 +33,10 @@ export class WalletScanner {
    * A snapshot of the accounts that have `scanningEnabled` set to true. Used
    * to tell what accounts should be scanned, and from what block.
    */
-  private scanningAccounts = new Array<{ account: Account; scanFrom: HeadValue | null }>()
+  private scanningAccounts = new Array<{
+    account: Account
+    scanFrom: { sequence: number; hash?: Buffer } | 'cursor'
```

Comment:
Does `'cursor'` mean that `scanFrom` is equal to the head of the wallet scanner?

Reply:
yea, any thoughts on better name?